script.js

```diff
@@ -1,4 +1,45 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var computerChoice = assignComputerChoice(getComputerChoice());
```

Comment:
it is better to define your function before you use them, getComputerChoice and assignComputerChoice should be defined before this line